src/lib.rs

```diff
@@ -121,14 +123,20 @@ pub mod pipeline {
             let mut in_agg = false;
             let mut pre_agg: Vec<Box<dyn operator::UnaryPreAggOperator>> = Vec::new();
             let mut post_agg: Vec<Box<dyn operator::AggregateOperator>> = Vec::new();
-            let mut op_iter = query.operators.into_iter().peekable();
+            let mut op_deque = query.operators.into_iter().collect::<VecDeque<_>>();
             let mut has_errors = false;
-            while let Some(op) = op_iter.next() {
+            while let Some(op) = op_deque.pop_front() {
                 match op {
                     Operator::RenderedAlias(rendered_alias) => {
                         // TODO: create a new QueryContainer here and parse the templated string
                         // -> expecting only Inline operators?
                         // let inner_query = QueryContainer::new(rendered_alias.value, ???);
+                        let (_span, operators) =
+                            operator_list(Span::new(CompleteStr(&rendered_alias.value)))
+                                .expect("alias parsing aways works");
+                        for new_op in operators {
+                            op_deque.push_front(new_op);
```

Comment:
I'm pretty sure this actually reverses the operators in the alias. 

`* | some_alias` where `some_alias` = `json | count(xyz)` => `[json, count(xyz]`

```
op_deque.push_front(json) // head is now `json`
op_deque.push(front(zyz) // head is now `xyx`
```

Reply:
I think all that's needed is failing test and then `operators.rev()`

Reply:
👍 yeah, good point - fixed + added new test.

I wanted to make the alias used for the test conditional, but wasn't able to because it's an integration test (no cfg!(test), etc.).

One potential solution is to implement loading aliases from a file, then we can remove the alias from the published version.